<launch>
  <node name="sensors" pkg="rosserial_python" type="serial_node.py" output="screen">
  <param name="port" value="/dev/ttyUSB0"/>
  <param name="baud" value="57600"/>
  </node>
  <node name="steppers" pkg="rosserial_python" type="serial_node.py" output="screen">
  <param name="port" value="/dev/ttyUSB1"/>
  <param name="baud" value="57600"/>
  </node>
  <node name="gui" pkg="scara_like" type="gui.py" output="screen"/>
  <node name="sensor_data_publisher" pkg="scara_like" type="sensor_data_publisher.py" output="screen"/>
  <node name="angle_target" pkg="scara_like" type="angle_target.py" output="screen"/>
  <node name="trajectory_plan" pkg="scara_like" type="traj_plan.py" output="screen"/>
  
</launch>